index.html

```diff
@@ -174,18 +250,63 @@ <h5 id="header-5">About me</h5>
                     </ul>
                 </li>
             </ul>
+            <div class="desktop-list">
+                <div class="sub-list">
+                  <div class="img-shape">
+                    <img src="images/1.png" alt="first shape">
+                    <h5 class="head" >Language</h5>
+                  </div>
+                  <ul class="list">
+                    <li>Javascript</li>
+                    <li>Ruby</li>
+                    <li>Html</li>
+                    <li>Css</li>
+                  </ul>
+                </div>
+                <div class="sub-list">
+                  <div class="img-shape">
+                    <img src="images/2.png" alt="second shape">
+                    <h5 class="head" >Frameworks</h5>
+                  </div>
+                  <ul class="list">
+                    <li>Bootstrap</li>
+                    <li>Ruby on Rails</li>
+                    <li>RSPec</li>
```

Comment:
It seems you limited the width for this section so please make sure to update this part of your page.

![image](https://user-images.githubusercontent.com/34779928/194727198-358c6110-24ce-46d7-abd9-b36648d605c7.png)

Reply:
# Implemented

![4th](https://user-images.githubusercontent.com/87594100/194728090-f5bb917c-0e23-498a-8c43-baa0b84ff183.png)